WIP: Test new compilers

h-vetinari opened the pull request:
Trying out what summer 2026 will feel like (GCC 15 & Clang 21 as default)

h-vetinari commented:
Closing as obsolete

conda-forge-admin commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipe/meta.yaml```) and found it was in an excellent condition.

I do have some suggestions for making it better though...


For **recipe/meta.yaml**:

 * ℹ️ The recipe is not parsable by parser `conda-souschef (grayskull)`. This parser is not currently used by conda-forge, but may be in the future. We are collecting information to see which recipes are compatible with grayskull.
 * ℹ️ The recipe is not parsable by parser `conda-recipe-manager`. The recipe can only be automatically migrated to the new v1 format if it is parseable by conda-recipe-manager.


<sub>This message was generated by GitHub Actions workflow run [https://github.com/conda-forge/conda-forge-webservices/actions/runs/15960115564](https://github.com/conda-forge/conda-forge-webservices/actions/runs/15960115564). Examine the logs at this URL for more detail.</sub>

h-vetinari committed:
GCC 15, Clang 21

h-vetinari committed:
set _LIBCPP_DISABLE_AVAILABILITY on macos

h-vetinari committed:
MNT: Re-rendered with conda-smithy 2026.6.14 and conda-forge-pinning 2026.07.29.10.37.28

Other tools:
- conda-build 26.5.0
- rattler-build 0.72.0
- rattler-build-conda-compat 1.4.18
